.travis.yml

```diff
@@ -32,10 +33,16 @@ addons:
   apt:
     packages:
       - swig
+  firefox: latest
```

Comment:
I added the extra firefox versions to travis so this I could do in my spare time.